Python/DB_Rede.py

```diff
@@ -15,6 +15,32 @@ def Refresh_Or_Create_Tables():
     metadata = sql.MetaData()
     engine = sqlalchemy()
 
+    # To do:
```

Comment:
Essa foi a definição da tabela no banco, toda vez que for adicionar uma nova tabela tem de fazer uma definição com esse formato adaptado a estrutura da tabela que deseja gerar